_posts/2022-10-18-trino-summit-2022-teaser-2.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Trino Summit 2022: Federating humans and data"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
+---
+
+Trino has long been the de facto standard to querying large data sets over your
+cloud or on-prem storage, also known as data lakes. This Trino Summit's theme 
+instead will showcase Trino's other claim to fame: query federation. In essence,
+Trino is a query engine providing an access point that exposes ANSI SQL across 
+[multiple data sources]({{site.url}}/docs/current/connector/).
+
+I can't help but bring up the analogy of how Trino federates heterogeneous data
+while this Trino Summit will federate many of us in the community form all
+corners of the earth. It really brings an appreciation to the international
+reach this technology has and makes me look forward to future in-person events!
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
+is free and takes place on November 10th. Check out the details below!
+
+<!--more-->
+
+## Register for the summit
+
+Trino Summit is being held at the Commonwealth Club in San Francisco, CA.
```

Comment:
```suggestion
Trino Summit is held at the Commonwealth Club in San Francisco, CA.
```

Reply:
I'm going to go with "will be" or "will be held". If we just say "is held" that makes it sound like it's always held at this location.